config/routes.rb

```diff
@@ -0,0 +1,37 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
+  root 'homepages#index'
+
+  get '/homepages', to: 'homepages#index', as: 'homepages'
+  get '/drivers', to: 'drivers#index', as: 'drivers'
+  get '/drivers/new', to: 'drivers#new', as: 'new_driver'
+  post '/drivers', to: 'drivers#create'
+  get '/drivers/:id/edit', to: 'drivers#edit', as: 'edit_driver'
+  patch '/drivers/:id', to: 'drivers#update'
+  get '/drivers/:id', to: 'drivers#show', as: "driver"
+  delete '/drivers/:id', to: 'drivers#destroy', as: 'destroy_driver'
```

Comment:
You should have used `resources` for these:

```suggestion
  resources :drivers
```